code/modules/projectiles/projectile/bullets.dm

```diff
@@ -210,6 +210,30 @@
 	damage = 50
 	weaken = 1
 
+/obj/item/projectile/bullet/rifle/slugger
+	name = "slugger round"
+	damage = 80
+	stun = 3
+	weaken = 3
+	penetrating = 5
+	armor_penetration = 20
+	hitscan = TRUE
+	maiming = TRUE
```

Comment:
I do think that hitscan AND maiming AND stun AND weaken AND armor penetration is like, way too much.